orthoinference/src/test/java/org/reactome/orthoinference/StableIdentifierGeneratorTest.java

```diff
@@ -56,12 +59,26 @@ public void generateOrthologousStableIdTest() throws Exception {
         Mockito.when(mockOriginalInst.getAttributeValue("stableIdentifier")).thenReturn(mockStableIdentifierInst);
         Mockito.when(mockStableIdentifierInst.getAttributeValue("identifier")).thenReturn(identifier);
 
-        StableIdentifierGenerator.setSpeciesAbbreviation(fakeAbbreviation);
-        StableIdentifierGenerator.setAdaptor(mockAdaptor);
+        Mockito.when(mockAdaptor.fetchInstanceByAttribute("StableIdentifier", "identifier", "=", "R-ABC-123456")).thenReturn(mockInstanceCollection);
 
         PowerMockito.when(InstanceUtilities.createNewInferredGKInstance(mockStableIdentifierInst)).thenReturn(mockOrthoStableIdentifierInst);
+        stIdGenerator.generateOrthologousStableId(mockInferredInst, mockOriginalInst);
+    }
 
-        StableIdentifierGenerator.generateOrthologousStableId(mockInferredInst, mockOriginalInst);
+    @Test
+    public void generateOrthologousStableIdRuntimeExceptionTest() throws Exception {
```

Comment:
For testing an exception, you can use this pattern:
```
@Rule
public ExpectedException expectedException = ExpectedException.none();

@Test
public void generateOrthologousStableIdRuntimeExceptionTest() {
    expectedException.expect(RuntimeException.class);  // only passes with this type of exception
    expectedException.expectMessage("Could not find stable identifier instance"); // checks the exception contains this message
    stIdGenerator.generateOrthologousStableId(mockInferredInst, mockOriginalInst);
}
```

Reply:
Yep I used this logic and adjusted it for the updated logging message